src/views/PeopleSearch/index.js

```diff
@@ -560,11 +577,29 @@ class PeopleSearch extends Component {
               label="Alumni"
             />
           ) : null}
+          <Media
+            query="(min-width: 960px)"
+            render={() => (
+              <Grid item xs={12} align="center">
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={this.state.displayLargeImage}
+                      onChange={() => {
+                        this.handleChangeDisplayLargeImages();
```

Comment:
```suggestion
                        this.setState({displayLargeImage: !this.state.displayLargeImage});
                        this.search();
```
This function is only called one time, and it happens to be here. It will only ever be called one time because of what it's doing, so we can put the two lines it contained in the onClick.

Reply:
I realize the function is only 2 lines, but so are half the other methods for handling changes in the search parameters. I was mirroring their structure which keeps the code in the JSX as clean as possible. I can still change this but I feel like even though it's only 2 lines, it would be better practice to stay consistent.

Reply:
Well, consistency is good. Discreteness is good too. Although, I like efficiency and I don't like reusability when it won't get reused. But it is a difference of philosophy so I'm just gonna resolve it.